### PR TITLE
Update full manifest.json example

### PR DESCRIPTION
**Pull request by adamquaile**
The example `manifest.json` from `symfony/framwork-bundle` is a little outdated, uses `make` etc..

| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -333,12 +334,11 @@ one used by ``symfony/framework-bundle``:
         },
         "gitignore": [
             ".env",
-            "/public/bundles/"
+            "/public/bundles/",
             "/var/",
             "/vendor/"
         ]
     }
-
```
Comment (xabbuh) [inline README.rst]: should be reverted

File: README.rst [hunk 2]
```diff
@@ -313,6 +313,7 @@ one used by ``symfony/framework-bundle``:
 
 .. code-block:: json
 
+
```
Comment (xabbuh) [inline README.rst]: should be reverted

**Comment by adamquaile** [comment]
@xabbuh Done.

**Comment by adamquaile** [comment]
@xabbuh Updated again, I will not rush using Github's browser editor without checking in future :)

**Pull request merged**